includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -70,10 +70,6 @@ public function init() {
 			return;
 		}
 
-		if ( WC_Payments_Features::is_stripe_ece_enabled() ) {
-			return;
-		}
-
```

Comment:
What is the purpose of this change? I'm not so sure this is required.

Reply:
I _think_ this was added because the AJAX endpoints weren't available? Since the merge they are now available with ECE specific parameters to I removed this change and re-added these lines of code.

Reply:
I already discarded these changes during the merge 🙂